### PR TITLE
Update tikv-configuration-file.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update TiKV configuration file about pessimistic transaction.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3178
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR. @CaitinChen

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1174,11 +1174,16 @@ Configuration items related to `import`
 
 ### `wait-for-lock-timeout`
 
-- The max time that a pessimistic transaction in TiKV waits for other transactions to release the lock, in milliseconds. If timed out, an error is returned to TiDB, and TiDB retries to add a lock. The lock wait timeout is set by `innodb_lock_wait_timeout`.
-- Default value: 1000
-- Minimum value: 1
+- The longest time that a pessimistic transaction in TiKV waits for other transactions to release the lock. If the time is out, an error is returned to TiDB, and TiDB retries to add a lock. The lock wait timeout is set by `innodb_lock_wait_timeout`.
+- Default value: 1s
```
Comment (CaitinChen) [inline tikv-configuration-file.md:1178]:
```suggestion
- Default value: 1 s
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1174,11 +1174,16 @@ Configuration items related to `import`
...
+- Minimum value: 1ms
```
Comment (CaitinChen) [inline tikv-configuration-file.md:1179]:
```suggestion
- Minimum value: 1 ms
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1174,11 +1174,16 @@ Configuration items related to `import`
...
 
 ### `wait-up-delay-duration`
 
-- When pessimistic transactions release the lock, among all the transactions waiting for lock, only the transaction with the smallest `start ts` is woken up. Other transactions will be woken up after `wait-up-delay-duration` milliseconds.
-- Default value: 20
+- When pessimistic transactions release the lock, among all the transactions waiting for lock, only the transaction with the smallest `start_ts` is woken up. Other transactions will be woken up after `wait-up-delay-duration`.
+- Default value: 20ms
```
Comment (CaitinChen) [inline tikv-configuration-file.md:1184]:
```suggestion
- Default value: 20 ms
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1174,11 +1174,16 @@ Configuration items related to `import`
...
+
+### `pipelined`
+
++ Enables the pipelined process of adding the pessimistic lock. With this feature enabled, after detecting that data can be locked, TiKV immediately notifies TiDB to execute the subsequent requests and write the pessimistic lock asynchronously, which reduces most of the latency and significantly improves the performance of pessimistic transactions.
```
Comment (CaitinChen) [inline tikv-configuration-file.md:1188]:
```suggestion
This configuration item enables the pipelined process of adding the pessimistic lock. With this feature enabled, after detecting that data can be locked, TiKV immediately notifies TiDB to execute the subsequent requests and write the pessimistic lock asynchronously, which reduces most of the latency and significantly improves the performance of pessimistic transactions.
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1174,11 +1174,16 @@ Configuration items related to `import`
...
 ### `wait-up-delay-duration`
 
-- When pessimistic transactions release the lock, among all the transactions waiting for lock, only the transaction with the smallest `start ts` is woken up. Other transactions will be woken up after `wait-up-delay-duration` milliseconds.
-- Default value: 20
+- When pessimistic transactions release the lock, among all the transactions waiting for lock, only the transaction with the smallest `start_ts` is woken up. Other transactions will be woken up after `wait-up-delay-duration`.
+- Default value: 20 ms
+
+### `pipelined`
+
+This configuration item enables the pipelined process of adding the pessimistic lock. With this feature enabled, after detecting that data can be locked, TiKV immediately notifies TiDB to execute the subsequent requests and write the pessimistic lock asynchronously, which reduces most of the latency and significantly improves the performance of pessimistic transactions.
```
Comment (TomShawn) [inline tikv-configuration-file.md:1188]:
```suggestion
This configuration item enables the pipelined process of adding the pessimistic lock. With this feature enabled, after detecting that data can be locked, TiKV immediately notifies TiDB to execute the subsequent requests and write the pessimistic lock asynchronously, which reduces most of the latency and significantly improves the performance of pessimistic transactions. 但有较低概率出现悲观锁异步写入失败的情况，可能会导致悲观事务提交失败。But there is a still low probability that the asynchronous write of the pessimistic lock fails, which causes failure of pessimistic transaction commit.
```

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1174,11 +1174,16 @@ Configuration items related to `import`
...
 ### `wait-up-delay-duration`
 
-- When pessimistic transactions release the lock, among all the transactions waiting for lock, only the transaction with the smallest `start ts` is woken up. Other transactions will be woken up after `wait-up-delay-duration` milliseconds.
-- Default value: 20
+- When pessimistic transactions release the lock, among all the transactions waiting for lock, only the transaction with the smallest `start_ts` is woken up. Other transactions will be woken up after `wait-up-delay-duration`.
+- Default value: 20 ms
+
+### `pipelined`
+
+This configuration item enables the pipelined process of adding the pessimistic lock. With this feature enabled, after detecting that data can be locked, TiKV immediately notifies TiDB to execute the subsequent requests and write the pessimistic lock asynchronously, which reduces most of the latency and significantly improves the performance of pessimistic transactions. But there is a still low probability that the asynchronous write of the pessimistic lock fails, which causes the failure of pessimistic transaction commit.
```
Comment (CaitinChen) [inline tikv-configuration-file.md:1188]:
```suggestion
This configuration item enables the pipelined process of adding the pessimistic lock. With this feature enabled, after detecting that data can be locked, TiKV immediately notifies TiDB to execute the subsequent requests and write the pessimistic lock asynchronously, which reduces most of the latency and significantly improves the performance of pessimistic transactions. But there is a still low probability that the asynchronous write of the pessimistic lock fails, which might cause the failure of pessimistic transaction commits.
```

**Pull request merged**